src/core-layers/point-cloud-layer/point-cloud-layer-vertex-64.glsl.js

```diff
@@ -47,17 +47,14 @@ void main(void) {
   vec2 projected_coord_xy[2];
   project_position_fp64(instancePositions64xy, projected_coord_xy);
 
-  vec2 vertex_pos_localspace[4];
-  vec4_fp64(vec4(positions.xy * radiusPixels / screenSize * 2.0, 0.0, 0.0), vertex_pos_localspace);
-
   vec2 vertex_pos_modelspace[4];
-  vertex_pos_modelspace[0] = sum_fp64(vertex_pos_localspace[0], projected_coord_xy[0]);
-  vertex_pos_modelspace[1] = sum_fp64(vertex_pos_localspace[1], projected_coord_xy[1]);
-  vertex_pos_modelspace[2] = sum_fp64(vertex_pos_localspace[2],
-    vec2(project_scale(instancePositions.z), 0.0));
+  vertex_pos_modelspace[0] = projected_coord_xy[0];
+  vertex_pos_modelspace[1] = projected_coord_xy[1];
+  vertex_pos_modelspace[2] = vec2(project_scale(instancePositions.z), 0.0);
   vertex_pos_modelspace[3] = vec2(1.0, 0.0);
 
   gl_Position = project_to_clipspace_fp64(vertex_pos_modelspace);
+  gl_Position += vec4(positions.xy * radiusPixels / viewportSize * 2.0, 0.0, 0.0);
```

Comment:
Nice catch!

I do recommend we use the `project_to_pixel`s function for this (I think that is already done in the 32 bit layer). It handles a number of cases like devicePixelRatio, adjusting `w` to match desired  `focalDistance` etc, and since the project module already calculates the necessary uniforms we could drop them from the point cloud layer JS code.

Reply:
Thanks, have fixed it. Thought that function was not injected to the 64 bit glsl shader, it actually works.

Reply:
I think the 64 bit project module imports the 32 bit one.